app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for single_car in cars:
+            if single_car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(single_car)
+                self.wash_single_car(single_car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = ((car.comfort_class
+                 * (self.clean_power - car.clean_mark)
+                 * self.average_rating)
+                 / self.distance_from_city_center)
```

Comment:
we recommend to follow the first option from checklist where we move all parameters on the next lines, so let's do it :)